bindings/python/crocoddyl/multibody/wrench-cone.cpp

```diff
@@ -0,0 +1,48 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2018-2020, University of Edinburgh
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+#include "python/crocoddyl/multibody/multibody.hpp"
+#include "crocoddyl/multibody/wrench-cone.hpp"
+
+namespace crocoddyl {
+namespace python {
+
+void exposeWrenchCone() {
+  bp::class_<WrenchCone>("WrenchCone", "Model of the friction cone as lb <= Af <= ub",
+                           bp::init<Eigen::Matrix3d, double, Eigen::Vector2d, bp::optional<std::size_t> >(
+                               bp::args("self", "rot", "mu", "box", "nf"),
```

Comment:
The reason why I used rotation matrix rather than normal vector is that wrench cone is a plane contact. I think wrench cone should be considered yaw rotation. Otherwise, I can make function with normal vector and yaw angle as inputs rather than rotation matrix. I will modify the function with your preference.